Formula/pick.rb

```diff
@@ -12,7 +12,6 @@ class Pick < Formula
   end
 
   def install
-    ENV["TERM"] = "xterm"
```

Comment:
The change was made inside of pick to remove the need to explicitly set `TERM` in https://github.com/calleerlandsson/pick/pull/181.

Hopefully, this isn't needed anymore.